client/src/pages/reports/Compact.js

```diff
@@ -0,0 +1,1012 @@
+  }
+  @media print {
+    position: fixed;
+    max-height: 70px;
+  }
+`
+
+const HeaderContent = styled.div`
+  ${HF_COMMON_STYLE};
+  top: 0mm;
+  border-bottom: 1px solid black;
+  background-color: ${props => props.bgc} !important;
+`
+
+const FooterContent = styled.div`
+  ${HF_COMMON_STYLE};
+  bottom: 0mm;
+  border-top: 1px solid black;
+  background-color: ${props => props.bgc} !important;
+`
+
+const PrintedByBox = styled.span`
+  align-self: flex-start;
+  width: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: flex-end;
+  flex-wrap: wrap;
+  font-size: 10px;
+  & > span {
+    display: inline-block;
+    text-align: right;
+  }
+`
+
+const ClassificationBanner = () => {
+  return (
+    <ClassificationBannerBox>
+      <CompactSecurityBanner />
+    </ClassificationBannerBox>
+  )
+}
+
+const ClassificationBannerBox = styled.div`
+  width: auto;
+  max-width: 67%;
+  text-align: center;
+  display: inline-block;
+  & > .banner {
+    padding: 2px 4px;
+  }
+`
+const CompactTable = ({ children }) => {
+  return (
+    <CompactStyledTable>
+      <thead>
+        <tr>
+          <SpacedTd colSpan="2" />
+        </tr>
+      </thead>
+      <tbody>{children}</tbody>
+      <tfoot>
+        <tr>
+          <SpacedTd colSpan="2" />
+        </tr>
+      </tfoot>
+    </CompactStyledTable>
+  )
+}
+
+CompactTable.propTypes = {
+  children: PropTypes.node
+}
+
+const CompactStyledTable = styled.table`
+  width: 100%;
+`
+
+const SpacedTd = styled.td`
+  height: 80px;
+`
+
+export const ROW_TYPES = {
+  titleLike: "titleLike",
+  onlyData: "onlyData"
+}
+
+export const CompactRow = ({ label, content, rowType, ...otherProps }) => {
+  const { style, className } = otherProps
+  const CustomStyled = styled(ROW_STYLE)`
+    ${style};
+  `
+  if (rowType === ROW_TYPES.titleLike) {
+    return (
+      <CustomStyled>
+        <RowLabel colSpan="2">{label}</RowLabel>
+      </CustomStyled>
+    )
+  }
+
+  if (rowType === ROW_TYPES.onlyData) {
+    return (
+      <CustomStyled>
+        <RowContent colSpan="2">{content}</RowContent>
+      </CustomStyled>
+    )
+  }
+
+  const lowerLabel =
+    typeof label === "string" ? label.toLocaleLowerCase() : label
+  const topHeader = className === "reportField"
+
+  return (
+    <CustomStyled className={className || null}>
+      <RowLabel topHeader={topHeader}>{lowerLabel}</RowLabel>
+      <RowContent>{content}</RowContent>
+    </CustomStyled>
+  )
+}
+
+CompactRow.propTypes = {
+  label: PropTypes.node,
+  content: PropTypes.node,
+  rowType: PropTypes.string
+}
+
+const ROW_STYLE = styled.tr`
+  vertical-align: top;
+  font-family: "Times New Roman", Times, serif;
+  width: 100%;
+`
+
+const RowLabel = styled.th`
+  padding: 4px 0;
+  font-style: italic;
+  color: grey;
+  max-width: 50%;
+  font-weight: 300;
+  width: ${props => (props.topHeader ? "15%" : "auto")};
+`
+
+const RowContent = styled.td`
+  padding: 4px 1rem;
+`
```

Comment:
Field labels and field values are not horizontally aligned with row, especially for custom fields. I suggest adding;
```
  /* fixes horizontal alignment for simple custom fields such as text, number, date */
  > div.form-control-static {
    padding-top: 0;
  }
  /* fixes horizontal alignment for complex custom fields such as array-of-objects */
  div.table-responsive > table > tbody > tr:first-of-type > td {
    padding-top: 0;
  }
```
These changes doesn't perfectly align everything but it is the smallest change with maximum gain and no/minimum side-effects I could come up with.